ui/main/src/app/business/templateGateway/cardTemplateGateway.ts

```
@@ -0,0 +1,213 @@
+
+export class CardTemplateGateway {
+    private static _card: Card;
+    private static _childCards: Card[];
+    private static _displayContext: string;
+    private static _entitiesAllowedToRespond: string[];
+    private static _entitiesUsableForUserResponse: string[];
+    private static _responseLocked: boolean;
+    private static _userAllowedToRespond: boolean;
+    private static _userMemberOfAnEntityRequiredToRespond: boolean;
+
+    private static _functionToDisplayLoadingSpinnner: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)